addons/TBMod_rhs/CfgVehicles.hpp

```diff
@@ -200,20 +202,54 @@ class CfgVehicles
         radarType = 8; // 0
     };
 
-    class Plane_CAS_01_dynamicLoadout_base_F;
-    class B_Plane_CAS_01_dynamicLoadout_F : Plane_CAS_01_dynamicLoadout_base_F // Vanilla A-10
+    class Plane_Base_F;
+    class Plane_CAS_01_base_F: Plane_Base_F
+    {
+        class Components;
+    };
+    class Plane_CAS_01_dynamicLoadout_base_F : Plane_CAS_01_base_F
+    {
+        class Components : Components
+        {
+            class TransportPylonsComponent
+            {
+                class Pylons
+                {
+                    class Pylons1;
+                    class Pylons2 : Pylons1
+                    {
+                        maxweight = 5000; // 500
```

Comment:
Sollte das so bleiben, dachte das war ein Test obs schwerer ist.

Reply:
Da hatte ich dich auf WhatsApp bereits zu befragt, weil ich nicht wusste, wie wir damit umgehen wollen. Am einfachsten ist es, die Passage wieder rauszunehmen, weil die keinen Sinn aktuell hat. Ich wollte das nur nicht tun, weil du da so lange dran gesessen hattest; das fühlte sich undankbar an. Wenn wir nicht vorhaben, das System zum Laufen zu bringen sollten wir es raus nehmen. Deine Entscheidung :)

Reply:
Wir habens jetzt in der Git History, also kann es raus soll, kann es raus, könnten es jederzeit zurücksetzen